test/end-to-end/paths/F-002_CreateNewSittingRecordsFeature_test.js

```diff
@@ -0,0 +1,127 @@
+const AddSittingRecordsPage = require('../pages/AddSittingRecordsPage');
+const ConfirmNewSittingRecordsPage = require('../pages/ConfirmNewSittingRecordsPage');
+const JudicialSittingRecordsPage = require('../pages/JudicialSittingRecordsPage');
+const ManageJudicialSittingRecordsPage = require('../pages/ManageJudicialSittingRecordsPage');
+
+Feature('Create new Sitting Records Feature Tests @functional @F-002');
+
+const randomDay = ('0' + (Math.floor(Math.random() * 28) + 1)).slice(-2);
+const randomMonth = ('0' + (Math.floor(Math.random() * 12) + 1)).slice(-2);
+
+Scenario('User is able to successfully save a single Sitting Record @S-002.1',({ I}) => {
+  I.loginWithJPSRecorderUser();
+  ManageJudicialSittingRecordsPage.addSittingRecordsInformation('Social Security and Child Support', 'Sutton', randomDay, randomMonth, '2022');
+  I.click('Continue');
+  JudicialSittingRecordsPage.clickAddSittingRecords();
+  I.createSittingRecord('Joe Ambrose', 'Tribunal Judge', 'Afternoon');
+});
+
+Scenario('User is successfully able to save multiple Sitting Records @S-002.2',({ I}) => {
+  I.loginWithJPSRecorderUser();
+  ManageJudicialSittingRecordsPage.addSittingRecordsInformation('Social Security and Child Support', 'Sutton', randomDay, randomMonth, '2022');
+  I.click('Continue');
+  JudicialSittingRecordsPage.clickAddSittingRecords();
+  AddSittingRecordsPage.selectJOH('Joe Ambrose', 'Tribunal Judge');
+  I.click('Add Another');
+  AddSittingRecordsPage.selectJOH('Mary', 'Regional Tribunal Judge', 1);
+  I.click('Add Another');
+  AddSittingRecordsPage.selectJOH('Brandon', 'Tribunal Member Disability', 2);
+  AddSittingRecordsPage.selectPeriod('Morning');
+  I.click('Continue');
+  ConfirmNewSittingRecordsPage.confirmSittingRecords('Joe Ambrose', 'Tribunal Judge', 'Morning', 'jps recorder');
```

Comment:
Repeated constants should be put into a variable. Avoid repeated hardcoded values